collectors/interrupts/data.go

```diff
@@ -0,0 +1,6 @@
+package interrupts
+
+type Interrupts struct {
+	Total  int64         // Total interrupts
+	PerCPU map[int]int64 // Number of interrupts for each cpu since system startup.
+}
```

Comment:
Data struct must be like 

type Interrupts struct {
	Total  InterrupstStat         // Total interrupts
	PerCPU map[int]InterrupstStat  // Number of interrupts for each cpu since system startup.
}

type InterrupstStat struct {
        Total int64 // The total number of interrupts
}